cloud/openstack/os_server.py

```diff
@@ -555,6 +560,7 @@ def main():
         boot_volume                     = dict(default=None, aliases=['root_volume']),
         terminate_volume                = dict(default=False, type='bool'),
         volumes                         = dict(default=[], type='list'),
+        scheduler_hints                 = dict(default=None),
```

Comment:
Good point, it expects arbitrary k/v pairs eg. `group: "<affinity group guid>"`. I will test and update accordingly